app/controllers/branding_groups_controller.rb

```diff
@@ -65,6 +65,17 @@ def update
         end
         # delete the param since it is not a real model param
         clean_params.delete("reset_#{image_name}")
+
+        # handle updates to the list of curators
+        curators = params[:curator_emails].split(',').map(&:strip)
+        user_ids = curators.map { |email| User.find_by(email: email)&.id }.compact
+        # ensure current user cannot remove accidentially remove themselves from the list
+        if @branding_group.users.include?(current_user)
+          user_ids << current_user.id unless user_ids.include?(current_user.id)
+        end
```

Comment:
consolidate the logic
```suggestion
        if @branding_group.users.include?(current_user) &&  !user_ids.include?(current_user.id)
          user_ids << current_user.id
        end
```